### ISSUE TITLE
Return keyboard focus to form errors on person form

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4044]
## Area or Component
Person form (add/edit person or self-registration)

## Quick Summary
The form doesn't return keyboard focus to the offending element in case of an erroring submission. Instead it scrolls to the top of the page. It should scroll to the first error (ideally after focusing the keyboard on the toast, but that's a separate problem)

### How to Replicate
1. Navigate to https://dev.simplereport.gov/app/register/E8Q2P (or another self-registration form in a lower environment)
2. Submit partial data, or invalid data (partial phone number, for instance)
3. Submit the form
4. Notice keyboard focus doesn't go back to the top error

## Accessibility Violation
https://dequeuniversity.com/class/forms/form-validation/focus-first-error

## Action Requested
Send focus to the first error on form submission.

If not already the case, we should also make sure to:
- Set invalid fields to aria-invalid="true".
- Ensure error messages are associated with form fields using aria-describedby so that screen reader users know how to fix the problem.
